api/test/test_helper/base_test_class.gd

```diff
@@ -128,6 +128,29 @@ func create_mock_asset_pack_state_with_status_dict(pack_name : String, status :
 		PlayAssetPackState._TRANSFER_PROGRESS_PERCENTAGE_KEY: 0
 	}
 
+func create_mock_asset_pack_state_dict():
```

Comment:
This generic ```create_mock_asset_pack_state_dict()``` is added so we don't need to repeatedly declare pack_name variables across functions like ```test_get_asset_pack_state_success()``` and ```assert_get_asset_pack_state_signal_is_success```